tools/ci_build/github/azure-pipelines/win-arm-crosscompile-ci-pipeline.yml

```diff
@@ -1,46 +1,41 @@
 jobs:
 - job: Windows_ARM_CrossCompile_CI_Dev
   timeoutInMinutes: 120
+  pool: Win-CPU-2019
   variables:
     buildDirectory: '$(Build.BinariesDirectory)'
+    ortPackageId: 'Microsoft.ML.OnnxRuntime'
+  strategy:
+    maxParallel: 2
+    matrix:
+      debug:
+        BuildConfig: 'Debug'
+      release:
+        BuildConfig: 'Release'
   steps:
     - template: templates/set-test-data-variables-step.yml
-    - task: UniversalPackages@0
-      displayName: 'Download python'
+    - task: UsePythonVersion@0
       inputs:
-        command: download
-        vstsFeed: '$(System.TeamProject)'
-        vstsFeedPackage: 'miniconda3_win64'
-        vstsPackageVersion: '4.5.11'
-        downloadDirectory: '$(Build.BinariesDirectory)\python'
-    - task: CmdLine@1
-      displayName: 'Run python installer'
-      inputs:
-        filename: '$(Build.BinariesDirectory)\python\installer.exe'
-        arguments: '/S /NoRegistry=1 /AddToPath=0 /RegisterPython=0 /D=$(Build.BinariesDirectory)\packages\python'
-      timeoutInMinutes: 10
+        versionSpec: '3.x'
     - task: BatchScript@1
       displayName: 'setup env'
       inputs:
         filename: '$(Build.SourcesDirectory)\tools\ci_build\github\windows\setup_env.bat'
         modifyEnvironment: true
         workingFolder: '$(Build.BinariesDirectory)'
-    - task: CmdLine@1
```

Comment:
The CMake version hosted in onnxruntime is too old and doesn't support VS 2019, so we're using the system provided one

Reply:
On other pipelines that are using the cmake_generator flag, the cmake_path flag is simply omitted. Are you sure we need to explicitly set the path to the system cmake?

Reply:
Probably not if it's in PATH, thanks for the hint